### DISCUSSION TITLE
Note in Readme or AIO Admin Panel on Fulltextsearch

### DISCUSSION THREAD

**Discussion by Fred-DTV** [nextcloud/all-in-one#3163] [Ideas] [answered]
Hey hey,

I've added the Fulltextsearch on our server and now it has been scanning files for over 4 days 😅
Which wouldn't be a problem, if it would run in the background, while the containers are running as normal, but it seems like it wants to finish all files before letting the nextcloud container boot completely.

I think it would be nice to have a little note or warning next to the container, which explains that this could take a long time if there are a lot of files for it to scan through and that the service won't run while it is doing so. Actually I could add it myself with a PR but I couldn't find the html file for the admin panel.

Or am I missing something here and the reason why nextcloud isn't starting is somewhere else?

P.S.: Luckily everyone at our office is on holidays anyways 😅

**Comment by szaimen** [comment, marked as answer]
> I think it would be nice to have a little note or warning next to the container, which explains that this could take a long time if there are a lot of files for it to scan through and that the service won't run while it is doing so.

Makes sense



> Actually I could add it myself with a PR but I couldn't find the html file for the admin panel.

You can adjust things here:

https://github.com/nextcloud/all-in-one/blob/main/php/templates/containers.twig#L553-L555

**Comment by Fred-DTV** [comment]
Just another update on this: It is day 6 and it is still scanning on our machine (16cores - 24gb Ram and 5TB of data) 🙈

**Reply by szaimen** [reply]
Yeah, 5TB seems like a LOT of data... As workaround you could potentially follow https://github.com/nextcloud/all-in-one/discussions/1709 and simply restart the nextcloud container which should stop it from indexing.

**Reply by Fred-DTV** [reply]
Yes thanks I had to do that, especially after I read an issue in the FTS repo where someone explained that it took two full weeks on his server.
I guess that's the end of that experiment for me.

**Reply by szaimen** [reply]
This is now released with v7.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
